test/graphql/queries/settings_query_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+module Queries
+  class SettingsQueryTest < GraphQLQueryTestCase
+    let(:query) do
+      <<-GRAPHQL
+      query {
+        settings {
+          totalCount
+          pageInfo {
+            startCursor
+            endCursor
+            hasNextPage
+            hasPreviousPage
+          }
+          edges {
+            cursor
+            node {
+              id
+              name
+            }
+          }
+        }
+      }
+      GRAPHQL
+    end
+
+    let(:data) { result['data']['settings'] }
+
+    test 'fetch settings' do
+      assert_empty result['errors']
+
+      expected_count = Setting.count
+      assert_not_equal 0, expected_count
```

Comment:
can you add also the tests from here: 
https://github.com/theforeman/foreman/blob/develop/test/controllers/api/v2/settings_controller_test.rb